citar-org.el

```diff
@@ -27,103 +27,102 @@
+    citar-org-activate-keymap)
   "List of activation functions for a citation.
 Each function takes one argument, a citation."
-  :group 'oc-bibtex-actions
+  :group 'citar-org
   :type '(repeat function))
 
 ;;; Keymaps
 
-(defcustom oc-bibtex-actions-map
+(defcustom citar-org-map
   (let ((map (make-sparse-keymap)))
-    (define-key map (kbd "o") '("open source (file or link)" . bibtex-actions-open))
-    (define-key map (kbd "e") '("open bibtex entry" . bibtex-actions-open-entry))
-    (define-key map (kbd "f") '("open source file" . bibtex-actions-open-library-files))
-    (define-key map (kbd "l") '("open source link" . bibtex-actions-open-link))
-    (define-key map (kbd "n") '("open notes" . bibtex-actions-open-notes))
-    (define-key map (kbd "r") '("refresh" . bibtex-actions-refresh))
+    (define-key map (kbd "o") '("open source (file or link)" . citar-open))
+    (define-key map (kbd "e") '("open bibtex entry" . citar-open-entry))
+    (define-key map (kbd "f") '("open source file" . citar-open-library-files))
+    (define-key map (kbd "l") '("open source link" . citar-open-link))
+    (define-key map (kbd "n") '("open notes" . citar-open-notes))
+    (define-key map (kbd "r") '("refresh" . citar-refresh))
     map)
   "Keymap for org-cite Embark minibuffer functionality."
-  :group 'oc-bibtex-actions
+  :group 'citar-org
   :type '(restricted-sexp :match-alternatives (keymapp)))
 
-(defcustom oc-bibtex-actions-buffer-map
+(defcustom citar-org-buffer-map
   (let ((map (make-sparse-keymap)))
-    (define-key map (kbd "o") '("open source (file or link)" . bibtex-actions-open))
-    (define-key map (kbd "e") '("open bibtex entry" . bibtex-actions-open-entry))
-    (define-key map (kbd "f") '("open source file" . bibtex-actions-open-library-files))
-    (define-key map (kbd "l") '("open source link" . bibtex-actions-open-link))
-    (define-key map (kbd "n") '("open notes" . bibtex-actions-open-notes))
-    (define-key map (kbd "r") '("refresh" . bibtex-actions-refresh))
+    (define-key map (kbd "o") '("open source (file or link)" . citar-open))
+    (define-key map (kbd "e") '("open bibtex entry" . citar-open-entry))
+    (define-key map (kbd "f") '("open source file" . citar-open-library-files))
+    (define-key map (kbd "l") '("open source link" . citar-open-link))
+    (define-key map (kbd "n") '("open notes" . citar-open-notes))
+    (define-key map (kbd "r") '("refresh" . citar-refresh))
     map)
   "Keymap for org-cite Embark at-point functionality."
-  :group 'oc-bibtex-actions
+  :group 'citar-org
   :type '(restricted-sexp :match-alternatives (keymapp)))
 
-(defcustom oc-bibtex-actions-citation-map
+(defcustom citar-org-citation-map
   (let ((map (make-sparse-keymap)))
-    (define-key map (kbd "<mouse-1>") '("default action" . bibtex-actions-dwim))
+    (define-key map (kbd "<mouse-1>") '("default action" . citar-dwim))
     (define-key map (kbd "<mouse-3>") '("embark act" . embark-act))
```

Comment:
```suggestion
    (with-eval-after-load 'embark
      (define-key map (kbd "<mouse-3>") '("embark act" . embark-act)))
```

Can I do this @minad?

And is it a problem that I get these warning using `with-eval-after-load`?

```elisp
Compiling file /home/bruce/Code/bibtex-actions/citar-org.el at Sun Oct 31 13:38:47 2021
citar-org.el:334:17: Warning: reference to free variable
    ‘embark-target-finders’
citar-org.el:334:17: Warning: assignment to free variable
    ‘embark-target-finders’
citar-org.el:335:17: Warning: reference to free variable ‘embark-keymap-alist’
citar-org.el:336:17: Warning: assignment to free variable
    ‘embark-keymap-alist’
citar-org.el:337:17: Warning: reference to free variable
    ‘embark-pre-action-hooks’
citar-org.el:337:17: Warning: assignment to free variable
    ‘embark-pre-action-hooks’
```

Reply:
You can declare this variables. Then then warnings will go away.

~~~ elisp
(defvar embark-pre-action-hooks)
...
~~~